File: 002.pl
#!/usr/bin/perl
use 5.012;
use warnings;

my ($fib1, $fib2) = (1, 1);
my $sum;

while ($fib1 < 4_000_000) {
    if (!($fib1 % 2)) {
       $sum += $fib1;
    }
    ($fib1, $fib2) = ($fib2, $fib1 + $fib2);
}

say $sum;

File: 002.py
#!/usr/bin/env python
#-*- coding:utf-8 -*-

fib1, fib2 = 1, 1
sum_fibs = 0
while fib1 < 4000000:
    if not fib1 % 2:
        sum_fibs += fib1
    fib1, fib2 = fib2, fib1 + fib2

print(sum_fibs)
